analytics.js

```diff
@@ -919,6 +929,7 @@ class Analytics {
    */
   load(writeKey, serverUrl, options) {
     logger.debug("inside load ");
+    this.options = options;
```

Comment:
Instead of caching the incoming options, how about creating the consent instance here?

Reply:
How will we get the response that needs to be sent through load?

Reply:
I see.